### PR TITLE
Popup javascript

### PR DESCRIPTION
**Pull request by Jolak5**
In this PR, I 

1. I used javascript to create the popup
2. I styled the popup to ensure it shows on both mobile and desktop version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -619,6 +703,144 @@ button:active {
...
+}
+
+.popup-section {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  position: absolute;
+  background-color: white;
+  transition: all 0.4s, top 0.4s;
+}
```
Comment (levy002) [inline styles.css]: - [ ] Kindly make sure your popup container is fixed-positioned. On your website, the pop-up is being opened but we can still scroll to see other parts of the page, however when the pop-up is opened we must not be able to see other parts of the page, we must only see the pop-up.

- [ ] Kindly make sure to add the background color on your popup container in to look like the provided Figma design. 

  - ### YOU POP-UP

   chrome-capture-2022-11-23 (4)

   chrome-capture-2022-11-23 (7)


  - ### FIGMA DESIGN

   image


  image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Jolak5  , :wave:
### I'm Levy, your PR reviewer

Excellent work :sunglasses: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to set up the `ES-LINT` linter correctly so it runs to check if there are no errors in your JavaScript code. I recommend you take a step back and read this[ lesson](https://github.com/microverseinc/linters-config/blob/master/html-css-js) to understand it clearly and follow the guideline to set it correctly.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @Jolak5, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Correct use of Github flow :heavy_check_mark: 

 ## Required Changes ♻️
- Since we started writing javascript code, kindly use the linter files from [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) as suggested by the previous reviewer. Please replace the `linters.yml` file with the new one so the `es-lint` checks can run.
Currently, Webhint, Stylelint, and Lighthouse checks are running.

image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @Jolak5  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
All linters are passing :heavy_check_mark: 
Fixed required issues :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**